pom.xml

```diff
@@ -95,6 +95,7 @@
 
     <!-- Release: enable publication to Bintray -->
     <artifactsToPublish>${project.groupId}:sonar-java-plugin:jar</artifactsToPublish>
+    <version.shade.plugin>3.1.0</version.shade.plugin>
```

Comment:
I think this one deserves a comment. At the very least to know when to remove it.